README.md

```diff
@@ -57,25 +57,31 @@ This guide assumes:
 
 ## Configuration
 
-Configuration is started in GitHub and completed in Mattermost. 
+GitHub plugin configuration starts by registering an OAuth app in GitHub and ends in Mattermost. 
 
 **Note:** If you're using GitHub Enterprise, replace all GitHub links below with your GitHub Enterprise URL.
 
 ### Step 1: Register an OAuth Application in GitHub
 
+You must first register the Mattermost GitHub Plugin as an authorized OAuth app regardless of whether you're setting up the GitHub plugin as a system admin or a Mattermost user.
+
 1. Go to https://github.com/settings/applications/new to register an OAuth app.
 2. Set the following values:
-   - **Application Name:** `Mattermost GitHub Plugin - <your company name>`
+   - **Application name:** `Mattermost GitHub Plugin - <your company name>`
    - **Homepage URL:** `https://github.com/mattermost/mattermost-plugin-github`
-   - **Authorization callback URL:** `https://your-mattermost-url.com/plugins/github/oauth/complete`, replacing `https://your-mattermost-url.com` with your Mattermost URL.
+   - **Authorization callback URL:** `https://your-mattermost-url.com/plugins/github/oauth/complete`, replacing `https://your-mattermost-url.com` with your Mattermost URL. This value needs to match the Mattermost server URL that you or your users users log in to. 
 3. Submit.
-4. Copy the **Client ID** and **Client Secret** in the resulting screen.
-5. Go to **System Console > Plugins > GitHub** and enter the **GitHub OAuth Client ID** and **GitHub OAuth Client Secret** you copied in a previous step.
-6. Hit **Save**.
+4. Click **Generate a new client secret** and provide your GitHub password to continue.
+5. Copy the **Client ID** and **Client Secret** in the resulting screen.
+6. Once you have successfully registered the Mattermost GitHub Plugin as an authorized OAuth app, switch to Mattermost and run `/github connect` in a Mattermost channel. You should receive a DM from the GitHub plugin about the features you have available.
```

Comment:
Are you sure hat works before 7. is complete?

Reply:
These steps are representative of what I experienced and completed on my first day as a MM staff member. I confirmed in the Plugin: GitHub channel that I didn't need to complete the final two steps as a user connecting to the MM Community Server. That being said, perhaps we can validate these additions the next time someone new is setting this up for the first time.

Reply:
In your case an admin already setup the OAuth app. Hence, you didn't had to follow step 1-5 anyway, right?

Reply:
That's my understanding, yes.

Reply:
Should we make this a HW issue - perhaps someone in the community would be open to validating the steps from both angles?